Create caller-links.json

ldonforth opened the pull request:
This file has the email address and website (if listed) of the callers in the Callers Directory at http://tinyurl.com/hnb72wv
it should start with a 
[
end with a 
]
and delete the comma after the final entry from the spreadsheet copy/paste

jeffkaufman commented on caller-links.json:
```diff
@@ -0,0 +1,171 @@
+[
+{"zip": "35747",  "name": "Jane Ewing", "email": "contradance@charter.net" },
+{"zip": "85022",  "name": "Michael Barraclough", "email": "michael@michaelbarraclough.com", "website": "michaelbarraclough.com"},
```
website links should start with `http://` or `https://` -- should I just assume `http://`?

jeffkaufman replied:
I'll go with assuming `http://`

ldonforth commented:
I didn't police how they were formatting. My guess is assuming http:// is
probably better.

I did remove one entry where the person had listed "none" for their webpage.

Do you want me to go through and add that in when it isn't there? Or you've
got that on your side?

On Wed, Feb 5, 2020 at 5:06 PM Jeff Kaufman <notifications@github.com>
wrote:

> Merged #16 <https://github.com/jeffkaufman/trycontra/pull/16> into master.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/jeffkaufman/trycontra/pull/16?email_source=notifications&email_token=AFZMPNV4O27NBE3LZRD53RLRBMZ73A5CNFSM4KQSVGSKYY3PNVWWK3TUL52HS4DFWZEXG43VMVCXMZLOORHG65DJMZUWGYLUNFXW5KTDN5WW2ZLOORPWSZGOWOCB25I#event-3011779957>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AFZMPNWBQPODAWQ5KODB3M3RBMZ73ANCNFSM4KQSVGSA>
> .
>

jeffkaufman commented:
I've now updated https://www.trycontra.com/callers-heatmap to use this file, and to show little cards when you click on the markers

ldonforth commented:
Thanks Jeff! Looks good :-)

I've added a link on my caller page, and I'll put up a social media post
about it.

On Tue, Feb 18, 2020 at 10:55 AM Jeff Kaufman <notifications@github.com>
wrote:

> I've now updated https://www.trycontra.com/callers-heatmap to use this
> file, and to show little cards when you click on the markers
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/jeffkaufman/trycontra/pull/16?email_source=notifications&email_token=AFZMPNTDG4D5MCRT4BTB3MDRDQAIBA5CNFSM4KQSVGSKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEMCQORA#issuecomment-587532100>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AFZMPNXMSBICQ6XIOGMMUKTRDQAIBANCNFSM4KQSVGSA>
> .
>
